libraries/platform/iot_network.h

```diff
@@ -94,6 +111,178 @@ typedef void ( * IotNetworkReceiveCallback_t )( IotNetworkConnection_t pConnecti
                                                 void * pContext );
 /* @[declare_platform_network_receivecallback] */
 
+/**
+ * @brief Provide an asynchronous notification of network closing
+ *
+ * A function with this signature may be set with @ref platform_network_function_setclosecallback
+ * to be invoked when the network connection is closed.
+ *
+ * @param[in] pConnection The connection on which data is available, defined by
```

Comment:
Should read: The connection that is being closed...

Reply:
good catch, fixed.